lkmpg.tex

```diff
@@ -925,6 +925,17 @@ \subsection{chardev.c}
 Don't worry if you don't see what we do with the data we read into the buffer; we don't do much with it.
 We simply read in the data and print a message acknowledging that we received it.
 
+In the multiple-threaded case, without any protection, concurrent access to the same memory may cause the race condition, and will not preserve the performance.
```

Comment:
We usually say, "in the multi-threaded environment / use case"

Reply:
Change "cause" to "lead to"

See https://lexically.net/TimJohns/Kibbitzer/revis024.htm
* `lead to` is not used with a human subject, and does not appear in the passive.
* `lead to` is less direct than `cause`, implying a series of steps between cause and effect.